decentraland/unity-renderer
# Fix/procedural skybox editor avatar color

## What does this PR change?

Issue Fixed: Skybox colors are affecting the editor avatar while in backpack.
Solution: New block for editor avatar color are introduced which will take effect when player is in backpack

## How to test the changes?

1. Go to: https://play.decentraland.org/?renderer=https://decentraland-olavra.s3.eu-west-1.amazonaws.com/branch/fix/procedural-skybox-editor-avatar-color
2. Go to backpack
3. Check avatar color (might be different from the environment.
4. Close the backpack
5. Change the time of the day to have a different environment color
6. Go back to backpack
7. Color of avatar in backpack should be same as before.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
